chapters/research-design.tex

```diff
@@ -1,288 +1,586 @@
+in response to an unexpected event.
+Intuitive knowledge of your project's chosen approach will make you
+much more effective at the analytical part of your work.
 \end{fullwidth}
 
-%------------------------------------------------
-
-\section{Counterfactuals and treatment effects}
-
-In causal analysis, a researcher is attempting to obtain estimates
-of a specific \textbf{treatment effect}, or the change in outcomes
-\index{treatment effect}
-caused by a change in exposure to some intervention or circumstance.\cite{abadie2018econometric}
-In the potential outcomes framework,
-we can never observe this directly:
-we never see the same person in both their treated and untreated state.\sidenote{\url{http://www.stat.columbia.edu/~cook/qr33.pdf}}
-Instead, we make inferences from samples:
-we try to devise a comparison group that evidence suggests
-would be identical to the treated group had they not been treated.
-
-This \textbf{control group} serves as a counterfactual to the treatment group,
-\index{control group}\index{treatment group}
-and we compare the distributions of outcomes within each
-to make a computation of how different the groups are from each other.
+%-----------------------------------------------------------------------------------------------
+%-----------------------------------------------------------------------------------------------
+
+\section{Causality, inference, and identification}
+
+When we are discussing the types of inputs -- ``treatments'' -- commonly referred to as
+``programs'' or ``interventions'', we are typically attempting to obtain estimates
+of program-specific \textbf{treatment effects}
+These are the changes in outcomes attributable to the treatment.\cite{abadie2018econometric}
+  \index{treatment effect}
+The primary goal of research design is to establish \textbf{causal identification} for an effect.
+Causal identification means establishing that a change in an input directly altered an outcome.
+  \index{identification}
+When a study is well-identified, then we can say with confidence
+that our estimate of the treatment effect would,
+with an infinite amount of data,
+give us a precise estimate of that treatment effect.
+Under this condition, we can proceed to draw evidence from the limited samples we have access to,
+using statistical techniques to express the uncertainty of not having infinite data.
+Without identification, we cannot say that the estimate would be accurate,
+even with unlimited data, and therefore cannot attribute it to the treatment
+in the small samples that we typically have access to.
+Conversely, more data is not a substitute for a well-identified experimental design.
+Therefore it is important to understand how exactly your study
+identifies its estimate of treatment effects,
+so you can calculate and interpret those estimates appropriately.
+All the study designs we discuss here use the \textbf{potential outcomes} framework
```

Comment:
add sidenote or index for potential outcomes framework

Reply:
Done as citation in 8e32d1dcfad3a6336f46eccfd992adc55515be31